### ISSUE TITLE
How to perform onClick event on card ?

### ISSUE THREAD

**Issue by sudikrt** [flschweiger/SwipeStack#44]
If i set onClickListener to the swipeStack then i cannot be able to swipe left or right?

**Comment by CallumCoombes** [comment]
I am having this same issue, was any solution found?

**Comment by sudikrt** [comment]
Yeah we have implemented the card click event  @CallumCoombes  use this 'com.geeksynergy:swipestack:1.0.1'
and it also contains bug it will cause crash when multi touch happens  
This bug also fixed..

**Comment by sudikrt** [comment]
we will update it to github soon @CallumCoombes

**Comment by CallumCoombes** [comment]
Oh awesome, bit confused as to how I add this to my project?

**Comment by sudikrt** [comment]
@CallumCoombes  Same thing what he has mentioned in his documentation only thing is
in app: Build.gradle add
 compile 'com.geeksynergy:swipestack:1.0.1'
instead of the library that he mentioned mentioned

**Comment by CallumCoombes** [comment]
Sadly I get 'failed to resolve: com.geeksynergy:swipestack:1.0.1'

**Comment by sudikrt** [comment]
wait
You need to add 
repositories {
        maven { url "http://dl.bintray.com/geeksynergy/GeekSupport" }
}
to your Project build.gradle

**Comment by CallumCoombes** [comment]
Ah of course!
Awesome, done that and I now have the onClick method, thanks so much!
